### PR TITLE
WIP Updated jobs app

### PR DESCRIPTION
**Pull request by assafg95**
First draft of the Job class with different fields. I still need to add more tests and check if all the methods work. This PR is without the Users app and is dependent on it. Once the User PR is merged I will insert it into this PR, so for now not all the checks might work.

### CODE REVIEW

**Comment by assafg95** [comment]
Will fix Flake8 errors tomorrow

**Review by AshkenYariv — CHANGES_REQUESTED** [review]
@assafg95 Looks really good!!! The jobs model looks great, the only thing I would add is tests for the new job function: the delete job and get_job_title as well as for the snippet_discription that it actually checks that there are 3 points (...) at the end of the sentence after snipping

**Review by shayakrach — CHANGES_REQUESTED** [review]
@assafg95 please add WIP to the title of the PR.

Also, add a message here to merge only after #28.

**Comment by assafg95** [comment]
Please merge this PR after PR #28 is merged

File: jobs/tests.py
Comment (assafg95) [inline jobs/tests.py:56]: Not completed still having problems with this test

**Comment by assafg95** [comment]
@shayakrach @AshkenYariv @yuvalk  @EdDev  Do you suggest that I add the User class so I can run my tests?

**Comment by shayakrach** [comment]
@assafg95 Please rebase on the updated master in your local branch and push it afterward, so we can see the changes.

**Comment by yuvalk** [comment]
why?
when closing, plz add a comment explaining why this PR is no longer needed

**Comment by yuvalk** [comment]
@assafg95 can you comment?

**Comment by assafg95** [comment]
There were a lot of conflicts in the rebase that were giving me problems. Also, anyways I wanted to change most of what I had there later. So to save time I decided to make a new branch from updated master.

**Comment by assafg95** [comment]
@yuvalk can I delete the branch if it is no longer relevant?

**Comment by yuvalk** [comment]
> There were a lot of conflicts in the rebase that were giving me problems. Also, anyways I wanted to change most of what I had there later. So to save time I decided to make a new branch from updated master.

I dont see anything here that warrent closing the PR, you could still reuse it
but even if that the case then you probably have a new PR ?

the best thing is to reference the new pr from the old one when closing it 
`closing this in favor of #XX`

**Comment by yuvalk** [comment]
> @yuvalk can I delete the branch if it is no longer relevant?

on your github fork? sure (and the local one of course)

**Comment by assafg95** [comment]
closing this in favor of #35

**Pull request closed without merging**
